pyproject.toml

```diff
@@ -21,7 +21,7 @@ redis = "^4.5.5"
 retry = "^0.9.2"
 transformers = "^4.30.2"
 streamlit = { version = "^1.23.1", optional = true }
-gTTS = { version = "^2.3.2", optional = true }
```

Comment:
Why not leave text-to-speech optional?

Reply:
When the tests run on pull request, it would fail due to unknown module gTTS

Reply:
Yeah, because I install with `-E podcast`, which installs the extra dependencies configured in the pyproject. However, if we mock things anyway, this could be one thing to mock. (It is not the expensive part of the test though.)

Reply:
Thanks. Currently, I added "install -E podcast" to ci.yml.